.github/workflows/run_static_checks.yaml

```diff
@@ -5,19 +5,35 @@ concurrency:
   cancel-in-progress: true
 
 on:
+  schedule:
+    - cron:  '0 */6 * * *'
   push:
     tags:
       - v*
       - e*
-  pull_request:
+  workflow_dispatch:
 
 jobs:
-    run_static_analysis:
-      runs-on: self-hosted
-      container: "ghcr.io/emqx/emqx-builder/5.0-3:24.1.5-3-alpine3.14"
-      steps:
-      - uses: actions/checkout@v2
-      - name: xref
-        run: make xref
-      - name: dialyzer
-        run: make dialyzer
+  run_static_analysis:
+    runs-on: ubuntu-20.04
```

Comment:
When using `self-host`, the matrix is executed sequentially, so I changed back to the `github host`

Reply:
that's because we have only one self-host runner